web/pgadmin/browser/server_groups/servers/static/js/server.ui.js

```diff
@@ -315,8 +316,16 @@ export default class ServerSchema extends BaseUISchema {
         }
       },{
         id: 'db', label: gettext('Maintenance database'), type: 'text', group: gettext('Connection'),
-        mode: ['properties', 'edit', 'create'], readonly: obj.isConnected, disabled: obj.isShared,
-        noEmpty: true,
+        mode: ['properties', 'edit', 'create'], readonly: obj.isConnected,
+        disabled: function(state) {
+          return !(isEmptyString(state.db_alias) && obj.isShared);
+        },
+      },{
+        id: 'db_alias', label: gettext('Database alias'), type: 'text', group: gettext('Connection'),
+        mode: ['properties', 'edit', 'create'], readonly: obj.isConnected,
+        disabled: function(state) {
+          return !isEmptyString(state.db);
+        },
```

Comment:
We should add help message here - `This is helpful in pg_bouncer where server can be connected using database alias name`